automatic/GoogleChrome/update.ps1

```diff
@@ -1,6 +1,8 @@
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = 'http://omahaproxy.appspot.com/all?os=win&amp;channel=stable'
+$paddedUnderVersion = '56.0.2925'
```

Comment:
Wouldn't it be better to have this also for future versions (assuming that Google won't change its practice)?